webpack.config.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 const webpack = require('webpack');
 const extend = require('extend');
 
-const production = process.env.NODE_ENV === 'production' ? false : true;
+const production = process.env.NODE_ENV === 'production' ? true : false;

```

Comment:
Remove the `? true : false`.
Should be just `const production = process.env.NODE_ENV === 'production';`